dvclive/live.py

```diff
@@ -162,10 +162,10 @@ def make_summary(self):
             json.dump(summary_data, f, indent=4)
 
     def read_step(self):
-        if self.exists:
+        if Path(self.summary_path).exists():
```

Comment:
If `summary=False` we don't really support `resume=True` because we rely on the `step` value stored in the summary, we might need to raise exception on `__init__`.

read_step only supports either existing summary or being the first instantiation (with `resume=True`, and `summary=True`) where summary doesn't exist yet.

Reply:
> If `summary=False` we don't really support `resume=True` because we rely on the `step` value stored in the summary, we might need to raise exception on `__init__`.

I guess we should raise an exception or make summary required.